packages/inscription-view/src/components/widgets/tag/Tags.tsx

```diff
@@ -5,16 +5,13 @@ import { IvyIcons } from '@axonivy/ui-icons';
 import { useReadonly } from '@axonivy/ui-components';
```

Comment:
I would reuse the `Tags` component. But we may need another name for it? e.g. `MultiValueSelect`?

Reply:
I made already some UI changes to this component.
- Putting X at the end of the tag
- Putting `Add`at the end of the list. Feels a bit more natural now.

![image](https://github.com/user-attachments/assets/ced59f13-ee55-4188-8792-74c557744319)

![image](https://github.com/user-attachments/assets/9d8e30ff-366f-4780-a24f-ff3dcc3048a1)

I think it looks already really good. I would make `+` sign smaller and use the same color as for `x` ?